sda-commons-server-prometheus/src/test/java/org/sdase/commons/server/prometheus/health/HealthCheckMetricsCollectorTest.java

```diff
@@ -11,23 +11,34 @@
 import java.util.List;
 import java.util.SortedMap;
 import java.util.TreeMap;
+import org.junit.Before;
 import org.junit.Test;
 
 public class HealthCheckMetricsCollectorTest {
 
-  @Test
-  public void shouldExposeMetricsForHealthChecks() {
-    HealthCheckRegistry registry = mock(HealthCheckRegistry.class);
-    HealthCheckMetricsCollector collector = new HealthCheckMetricsCollector(registry);
+  private HealthCheckMetricsCollector collector = null;
+  private HealthCheckRegistry registry = null;
+
+  @Before
+  public void setUp() {
+    registry = mock(HealthCheckRegistry.class);
+    collector = new HealthCheckMetricsCollector(registry);
+  }
 
+  private void initHealthcheckRegistryMock(int numHealthy, int numUnhealthy) {
     SortedMap<String, Result> results = new TreeMap<>();
-    results.put("healthy_check", Result.healthy());
-    results.put("unhealthy_check", Result.unhealthy("Something went wrong"));
+    for (int i = 0; i < numHealthy; i++) results.put("healthy_check_" + i, Result.healthy());
+    for (int i = 0; i < numUnhealthy; i++)
+      results.put("unhealthy_check_" + i, Result.unhealthy("Something went wrong"));
```

Comment:
Please always use braces (see https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used). You might want to run `./gradlew spotlessApply` which should add it automatically.

Reply:
Thank you!
I updated the pull request, used the braces, adjusted the README, renamed the commit to comply to the semantic commit messages and rebased on current master branch.
A final `./gradlew spotlessApply`ended with _build successful_.